### DISCUSSION TITLE
Update data file

### DISCUSSION THREAD

**Discussion by mgiardinelli** [msg-systems/coreferee#28] [General]
Hi Richard, nice job with coreferee. If I want to update the data files like (https://github.com/msg-systems/coreferee/blob/master/coreferee/lang/en/data/exclusively_person_words.dat) will I need to re-train the model(s)?

**Comment by richardpaulhudson** [comment]
Hi Mike, essentially no: these data files are used by the rules-based component within Coreferee rather than the machine-learning component. Having said that, the rules are applied during training as well as during normal operation, so if you make very extensive changes to the rules it might be worth retraining.
